### DISCUSSION TITLE
Running python script

### DISCUSSION THREAD

**Discussion by noemiLuna** [Tyrrrz/CliWrap#195] [Q&A]
Hi, I'm trying to run a python script from command line. 
I was able to do it without CliWrap as following reported:

 public void ExecutePythonCommand()
    {
        Process p = new Process();
        ProcessStartInfo startInfo = new ProcessStartInfo();
        startInfo.WindowStyle = ProcessWindowStyle.Hidden;
        startInfo.FileName = "cmd.exe";
        startInfo.Arguments = "/c \"python \"C:\\Users\\python_scripts\\__main__.py\"";
        p.StartInfo = startInfo;
        p.Start();
    }

With CliWrap I'm not able to understand which is the proper way to run it. I tried for example the following codes:
example1:
  var result = await Cli.Wrap("cmd.exe")
           .WithArguments(arguments: "python C:/Users/python_scripts/__main__.py")
           .ExecuteBufferedAsync();

example2:
var result = Cli.Wrap("cmd.exe")
            .WithArguments("python __main__.py")
            .WithWorkingDirectory("C:\\Users\\python_scripts")
            .ExecuteAsync();

It would be great if someone could clarify if I'm doing some mistake. Thank you in advance!

**Comment by Tyrrrz** [comment]
You're missing the `/c` argument to `cmd` in both cases, but ultimately you shouldn't need it. This should work:

```csharp
var result = await Cli.Wrap("python")
	.WithArguments(new[] { "C:/Users/python_scripts/main.py" })
	.ExecuteBufferedAsync();
```
